mir-module/src/main/resources/META-INF/resources/js/mir/ror-search.js

```diff
@@ -1,148 +1,226 @@
-const RORSearch = {
-    baseURL: "https://api.ror.org/",
+/**
+ * Initializes ROR-based affiliation autocomplete with Select2
+ * in XEditor forms, with i18n support and fallback messages.
+ */
+$(document).ready(async function () {
```

Comment:
With a view to the Bootstrap 5 migration and future jquery replacement, it should be carefully checked whether new jquery code is necessary or whether this cannot be solved with standard javascript.

Reply:
I removed invocation of `$(document)` but select2 itself relies on jquery